node.gypi

```diff
@@ -256,20 +239,13 @@
         }],
       ],
     }],
-    [ 'node_engine=="v8"', {
+    ['node_engine=="chakracore"', {
```

Comment:
>node_engine [](start = 6, length = 11)

can you try building v8 as well? There is some known float64 issue, but with your change, we should atleast not see more issues.